database/index.js

```diff
@@ -0,0 +1,72 @@
+/*
+- create a folder then change path in the terminal/cmd to that folder and execute the command 'npm install mysql'
+- create an 'index.js' file in the same folder and copy the following code in it.
+- zip the folder and upload it to AWS Lambda.
+*/
+// Following code is based on the assumption that the table structure consists of a column named 'last_modified(timestamp)' on the basis of which incremental update will happen
+// and 'isDeleted(int11)' whose value defines if that particular row has been deleted or not (0 - not deleted, 1 - deleted).
+var mysql = require('mysql');
+exports.handler =  (request, context, callback) => {
+  var pool  = mysql.createPool({
+    host     : "host",
+    port     : "port",
+    user     : "user_name",
+    password : "password",
+    database : "db"
+  });
+  //prevent timeout from waiting event loop
+  context.callbackWaitsForEmptyEventLoop = false;
+  pool.getConnection(function(err, connection) {
+    // Use the connection
+    var query = "";
+    if(request.state.employee === undefined || request.state.employee === null) {
+        query = 'SELECT * from table_name order by last_modified';
+    }
+    else {
+        query = 'SELECT * from table_name where last_modified >= "'+request.state.table_name+'" order by last_modified';
+    }
+    connection.query(query, function (error, results, fields) {
+      // And done with the connection.
+      connection.release();
+      // Handle error after the release.
+      if (error) callback(error);
+      else {
+        var insertTransactions = [];
+        var deleteTransactions = [];
+        var cursor;
+        for (i in results) {
+          // store the record as a key value pair of attribute name and attribute value
+          // 'last_modified' is the column on the basis of which incremental update will happen
+          // 'isDeleted' defines if teh column has been deleted or not
+          var a = { "column_1": results[i].column_1, "column_2": results[i].column_2, "column_3": results[i].column_3,"isDeleted":results[i].isDeleted,"last_modified":results[i].last_modified }
+          console.log(a);
+          if(results[i].isDeleted===0){
+          insertTransactions.push(a);
```

Comment:
poor formatting for sample code for customers, plz take care of proper indentation and spacing